src/Symfony/Bridge/Twig/Node/FormThemeNode.php

```diff
@@ -28,7 +28,7 @@ public function compile(Compiler $compiler)
     {
         $compiler
             ->addDebugInfo($this)
-            ->write('$this->env->getRuntime(\'Symfony\Bridge\Twig\Form\TwigRenderer\')->setTheme(')
+            ->write('$this->env->getRuntime(\'Symfony\Component\Form\FormRenderer\')->setTheme(')
```

Comment:
hat about doing the check at compile time work? ie using `$compiler->getEnvironment()->getRuntime()`?